include/deal.II/matrix_free/fe_evaluation_data.h

```diff
@@ -735,8 +735,16 @@ class FEEvaluationData
   const Point<dim, Number> *quadrature_points;
 
   /**
-   * A pointer to the Jacobian information of the present cell. Only set to a
-   * useful value if on a non-Cartesian cell.
+   * A pointer to the inverse transpose Jacobian information of the present
+   * cell. Only set to a useful value if on a non-Cartesian cell. If the cell is
```

Comment:
What do you mean by `Only set to a useful value if on a non-Cartesian cell.`? Do you mean that the index only goes by the quadrature points `q` for the non-affine case, while for affine cells we always need to index with `[0]`? And that the affine/Cartesian case also has the actual Jacobian in the same location?

Reply:
This was something that was already there and frankly something I too was confused about, but something I left untouched. I'll clearify it with that for affine/Cartesian cells only index 0 is set.